### DISCUSSION TITLE
Timing of Public Facing Files - is there expectations for the files to be published on given date or will there be various dates?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#54] [Q&A] [answered]
Timing of Public Facing Files - is there expectations for the files to be published on given date or will there be various dates?  The mandate indicates that Plans and issuers are required to make these files public for plan or policy years beginning on or after January 1, 2022, so does that mean that the expectations are to process the files at the end of the month to make them available by  first of the month?    Would the publish dates need to be the first of the month or the last of the month?  For January, would the first files last_updated_on date  be expected for 1/1/2022 or  by 1/31/2022?

**Comment by shaselton-usds** [comment, marked as answer]
@tdfow Trying to circle around and close up some old threads. We were able to [clarify the timing question](https://github.com/CMSgov/price-transparency-guide#timing-updates-for-machine-readable-files) that has been floating around here (and on the webinar as well). 

The short of it is: CMS is flexible on the timing of producing the files on a monthly cadence, just try to keep close to 30 days. There isn't a specific day of the month in which the files must be made public.

**Reply by tdfow** [reply]
@shaselton-usds so for the 7/1 date, does that mean the timing can be any day in July, then keep the monthly cadence close to every 30 days?

**Reply by shaselton-usds** [reply]
@tdfow take a peek at: https://github.com/CMSgov/price-transparency-guide/discussions/136#discussioncomment-2569507
